lib/octicons_helper/lib/octicons_helper/helper.rb

```diff
@@ -2,13 +2,23 @@
 require "action_view"
 
 module OcticonsHelper
-
   include ActionView::Helpers::TagHelper
 
+  mattr_accessor :octicons_helper_cache, default: {}
+
   def octicon(symbol, options = {})
     return "" if symbol.nil?
 
-    icon = Octicons::Octicon.new(symbol, options)
-    content_tag(:svg, icon.path.html_safe, icon.options) # rubocop:disable Rails/OutputSafety
+    cache_key = [symbol, options].to_s
+
+    if octicons_helper_cache[cache_key]
+      octicons_helper_cache[cache_key]
+    else
+      icon = Octicons::Octicon.new(symbol, options)
```

Comment:
Do you think we should include any of the raise conditions here when the symbol doesn't exist?

Reply:
@jonrohan Good idea! I think we should look into it in another PR.